From fe7ce8143687d0bebc727c8d2f97b14f6e5e75ee Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:02 -0400
Subject: [redhat] kabi: Adding symbol mapping_tagged (mm/page-writeback.c)

Message-id: <be9273959353e4fdd5211ecba52ea986575336ca.1410510773.git.poros@redhat.com>
Patchwork-id: 96098
O-Subject: [PATCH RHEL7.1 122/233] kabi: Adding symbol mapping_tagged (mm/page-writeback.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Used to search the address_space radix tree if any pages that are marked with specified tag are present.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/mapping_tagged
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/mapping_tagged
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/mapping_tagged

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/mapping_tagged b/redhat/kabi/kabi-module/kabi_ppc64/mapping_tagged
new file mode 100644
index 0000000..7763f9f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/mapping_tagged
@@ -0,0 +1,2 @@
+#1-
+0xf5b710db mapping_tagged vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/mapping_tagged b/redhat/kabi/kabi-module/kabi_s390x/mapping_tagged
new file mode 100644
index 0000000..d044d21
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/mapping_tagged
@@ -0,0 +1,2 @@
+#1-
+0x3cff268a mapping_tagged vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/mapping_tagged b/redhat/kabi/kabi-module/kabi_x86_64/mapping_tagged
new file mode 100644
index 0000000..39ca090
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/mapping_tagged
@@ -0,0 +1,2 @@
+#1-
+0xabee204a mapping_tagged vmlinux EXPORT_SYMBOL
-- 
1.7.1